src/services/handlers/searchApi.ts

```diff
@@ -23,7 +24,7 @@ const getSearchResults = async (searchFieldsObject: ISearchPayload): Promise<ISe
 
 const getSearchResultsCount = async (searchFieldsObject: ISearchPayload): Promise<{ totalResults: number }> => {
   try {
-    const payload = buildSearchQuery(searchFieldsObject);
+    const payload = buildSearchQuery(searchFieldsObject, [], true, true);
     if (payload.query.bool.must?.length) {
       const response = await elasticSearchClient.post(elasticSearchAPIPaths.countPath, payload);
       const data = await response.data;
```

Comment:
what if the count is not available in data. should the line below like data?.count??0

Reply:
To be honest, the elasticsearch standard response format will always have a count attribute when requesting a count of only one document. If no documents are present for the search criteria, it will always send `0`. Here `?` It's not necessary. However, we can send `0` as requested.